PatientRegistrationApp/src/App.vue

```diff
@@ -228,36 +228,55 @@ import SinglePatientReviewSubmit from './components/SinglePatientReviewSubmit';
 
 export default {
 	name: 'App',
+	components: 
+	{
+		GreetingPage,
+		SinglePatientHomeAddress,
+		SinglePatientContactInfo,
+		SinglePatientPersonalInfo,
+		SinglePatientEmergencyContact,
+		SinglePatientReviewSubmit,
+	},
 	methods: 
 	{
 		submit()
 		{
 			alert('You clicked submit!')
 		},
+		goToPage(pageNum)
+		{
+			this.page=pageNum
+		},
+		verifySinglePatientHomeAddress()
+		{
+			this.$refs.singlepatienthomeaddress.verifyFormContents() ? this.goToPage(3) : this.goToPage(2);
+		},
+		verifySinglePatientContactInfo()
+		{
+			this.$refs.singlepatientcontactinfo.verifyFormContents() ? this.goToPage(4) : this.goToPage(3);
+		},
+		verifySinglePatientPersonalInfo()
+		{
+			this.$refs.singlepatientpersonalinfo.verifyFormContents() ? this.goToPage(5) : this.goToPage(4);
+		},
+		verifySinglePatientEmergencyContact()
+		{
+			this.$refs.singlepatientemergencycontact.verifyFormContents() ? this.goToPage(6) : this.goToPage(5);
+		},	
 		setSinglePatientRegistration()
 		{
 			this.registrationPath = 1;
```

Comment:
Feel like this would also be a great example of being more explicit like 1 should be defined as REGISTRATIONPATH.SINGLE_PATIENT for example...